platform/src/test/java/platformstepdefinition/PF_CreateAccount.java

```diff
@@ -90,4 +92,50 @@ public void enters_confirmation_code() throws Throwable {
 
 	}
 
+	/**
+	 * @Author : Chetan Sonparote
+	 * @Date : 6 Aug 2021
+	 * @Description: step def for resend code functionality
+	 */
+
+	@Then("^Validate that new code is accepted$")
+	public void validate_that_new_code_is_accepted() throws Throwable {
+		// throw new PendingException();
+		CreateAccountStep3 createAccountStep3 = new CreateAccountStep3(driver);
+		Assert.assertTrue(CommonFunctions.isElementVisible(createAccountStep3.passwordField));
+
+	}
+
+	@And("^User clicks resend confirmation code$")
+	public void user_clicks_resend_confirmation_code() throws Throwable {
+		// throw new PendingException();
+		CreateAccountStep2 createAccountStep2 = new CreateAccountStep2(driver);
+		createAccountStep2.clickResendCodeLink();
+		NewBaseClass.validateError(null, "確認コードを再送しました、宛先メールをご確認ください。");
+
+	}
+
+	/*
+	 * @And("^enters new confirmation code$") public void
+	 * enters_new_confirmation_code() throws Throwable { // throw new
+	 * PendingException(); CreateAccountStep2 createAccountStep2 = new
+	 * CreateAccountStep2(driver); createAccountStep2.addConfirmationCode(); }
+	 */
+
+	@Then("^Validate that error is displayed for invalid code$")
+	public void validate_that_error_is_displayed_for_invalid_code() throws Throwable {
+		// throw new PendingException();
+		NewBaseClass.validateError(null, "6桁の確認コードを入力してください。");
```

Comment:
Please do not use the content. You can use it through excel.

Reply:
I am validating error message here. Cud not find error string in excel

Reply:
done